pyaedt/application/Design.py

```diff
@@ -1347,6 +1347,7 @@ def activate_variable_sensitivity(self, variable_name, min_val=None, max_val=Non
         """
         arg = ["NAME:AllTabs"]
         self._optimetrics_variable_args(arg, "Sensitivity",variable_name, min_val, max_val)
+        print(arg)
```

Comment:
We really should be using the logger here.

Reply:
removed. it was in my debug phase